python/observability-online-eval/pydantic-ai/advanced_agent.ipynb

```
@@ -0,0 +1,239 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Initializing Maxim instrumentation for Pydantic AI...\n",
+      "Instrumentation complete!\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Instrument Pydantic AI once at the top\n",
```

Ruff (0.12.2)

Missing return type annotation for public function `run_advanced_example`

Add return type annotation: `None`

(ANN201)